src/hooks/index.ts

```diff
@@ -32,3 +32,4 @@ export {
   useSetColumnDefaultSortOrder,
   useLegacyUIURL,
 };
+export { useAnalytics } from "./useAnalytics";
```

Comment:
Can you export it the same way the rest of the exports in this file are structured.

Reply:
nope